docs/source/2020CAD.rst

```diff
@@ -55,6 +55,30 @@ Model Configuration and Datasets
    
       crontab 2020CAD.crontab  
 
+  .. group-tab:: SRW.v1.0
+
+    The case runs are initialized at 12z Feb 03, 2020 with 90 hours forecasting.The app uses ``config.sh`` to define the runtime settings. The settings that need to be modified to set up the first cycle, last cycle, forecast length and cycle hour are listed below. 
```

Comment:
in all places, I recommend to replace `with XX hours forecasting` with `with a forecast length of XX hours`.